src/components/MoneyRequestReportView/SearchMoneyRequestReportEmptyState.tsx

```diff
@@ -22,6 +22,7 @@ function SearchMoneyRequestReportEmptyState() {
                 headerStyles={[styles.emptyStateMoneyRequestReport]}
                 lottieWebViewStyles={styles.emptyStateFolderWebStyles}
                 headerContentStyles={styles.emptyStateFolderWebStyles}
+                minModalHeight={380}
```

Comment:
Could we add this as a CONST?

Reply:
we could, but I don't think it's that important. This component itself is a specific component dedicated to `MoneyRequestReport`. Also empty state inside it also uses a specific number as a default: https://github.com/Expensify/App/blob/e2e322f5b6dea77b48584781f60dad1151280454/src/components/EmptyStateComponent/index.tsx#L32

If I'm to make this a const, I would still keep it inside this file. I don't think is a value that anyone else should read or use. Its a specific height for this specific component